libs/components/indicators/src/lib/modules/alert/alert.component.html

```diff
@@ -15,7 +15,10 @@
   }"
 >
   <div aria-hidden="true" class="sky-alert-icon-theme-modern">
-    <sky-icon-stack [baseIcon]="alertBaseIcon" [topIcon]="alertTopIcon">
+    <sky-icon-stack
+      [baseIcon]="indicatorIcon.modernThemeBaseIcon"
+      [topIcon]="indicatorIcon.modernThemeTopIcon"
```

Comment:
This way of referencing the icons is what allows us it this way to remove the `updateAlertIcon()` helper function, the `ngOnInit()`, and another `public` property.